azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Improve quest Fields of Grief

## Changes Proposed:
- ~~Increase spawn locations from 10 -> 32 (let's make it look like a field and not a tiny garden with pumpkins)~~
-  ~~Reduce respawn time from 700s -> 120s~~
- ~~This has been taken from the current TC database (no other sources found)~~
- https://classicdb.ch/?item=2846#comments - says spawn around 5 mins, currently it's 700s
- https://classic.wowhead.com/quest=365/fields-of-grief#comments - 10 locations shown

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4528

Thanks to @Kitzunu and @nicoaravena for helping sorting things out.



## Tests Performed:
- ~~Applied SQL, noticed the many pumpkins~~
- Applied SQL, faster respawn of pumpkins



## How to Test the Changes:


- `.quest add 365`
- `.tele sollidenf`
- loot pumpkins, notice respawn is 5 minutes

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
